kpatch-build/create-diff-object.c

```diff
@@ -2562,9 +2562,12 @@ static void kpatch_create_patches_sections(struct kpatch_elf *kelf,
 
 	/* count patched functions */
 	nr = 0;
-	list_for_each_entry(sym, &kelf->symbols, list)
-		if (sym->type == STT_FUNC && sym->status == CHANGED && !sym->parent)
-			nr++;
+	list_for_each_entry(sym, &kelf->symbols, list) {
+		if (sym->type != STT_FUNC || sym->status != CHANGED ||
+		    sym->parent)
+			continue;
+		nr++;
```

Comment:
I don't really feel this is improving readability in this particular instance.
The change here just adds a statement rather than simplify overall indentation.

(I do agree the other change in this commit makes the code nicer to read).

Reply:
In isolation I would agree, but the benefit is that this makes the condition identical to the 2nd loop's conditional, so the reader can verify the conditions are the same and 'nr' is valid.

Reply:
I see, fair point.

Reply:
Initially I thought the same as @julien-thierry, but then the code copy part made sense.  I guess if it were copied again, an inline helper function/macro could made it easier on the eyes.

Reply:
Might as well add my tidbit, reviewing from a checked out copy of the file made the intention apparent.